.github/workflows/dispense_rewards.yml

```diff
@@ -73,3 +73,17 @@ jobs:
         if: github.event_name == 'schedule'
         run: |
           ./dftool dispense_passive 1 0 $date
+
```

Comment:
The filenames in this directory are confusing. Renaming this file from dispense_passive -> dispense_active makes it worse. We need to get things consistent.

Currently, here are the relevant files, and their function:
- dispense_passive.yml / dispense_rewards.yml : Passive DF & Predictoor DF OCEAN?
- flow.yml: says "Dispense Active Rewards" but I think it's for Volume DF & Challenge DF?
- predictoor_rose.yml: rewards for Predictoor DF ROSE? 

Perhaps what's better:
- dispense_passive.yml : Passive DF
- dispense_volume.yml: Volume DF & Challenge DF (we could have had "challenge" in the name. But we'll turn off "challenge" shortly anyway; therefore this minimizes changes)
- dispense_predictoor_ocean.yml : Predictoor DF OCEAN
- dispense_predictoor_rose.yml: Predictoor DF ROSE